.circleci/config.yml

```diff
@@ -42,13 +44,45 @@ jobs:
   validate:
     executor: deploy
     parameters:
-      tf_path:
+      root:
+        type: string
+    environment:
+      TF_LOG: info
+    steps:
+      - checkout
+      - run:
+          name: "Terraform validate"
+          command: |
+            for dir in ./examples/*/; do
+              if [ -d "$dir" ]; then
+                echo "Processing $dir"
+                cd "$dir"
+                terraform init
+                terraform validate
+                cd - > /dev/null
+              fi
+            done
```

Comment:
We now validate all the examples: https://app.circleci.com/pipelines/github/CircleCI-Public/server-terraform/855/workflows/b0eecc14-f031-47b4-b7ad-35ae1170f2c6/jobs/3597?invite=true#step-102-0_28